client/src/components/PeopleView/EmployeeModal.js

```diff
@@ -63,13 +65,13 @@ export default class EmployeeModal extends React.Component {
     }
 
     render() {
-        const { e, handleClose, projects, sortedEmployees, handleSelectEmployee, orderedEmployees, date } = this.props;
+        const { e, handleClose, projects, sortedEmployees, handleSelectEmployee, orderedEmployees, entries, date } = this.props;
 
         if (!e) { return null; }
 
         const lastEntry = e.entry.size > 0 ? e.entry.get(-1) : null;
```

Comment:
This is done in a different way compared to EmployeeCard. Also since lastEntry, color and flagged are used in both, we could have a helper to get these values so that we can stay DRY.